quic/s2n-quic-core/src/crypto/tls.rs

```diff
@@ -58,7 +58,7 @@ pub trait Context<Crypto: crate::crypto::CryptoSuite> {
     /// Called when the client's application parameters are available, prior
     /// to completion of the handshake.
     ///
-    /// The `server_params` is provided as a mutable Vec<u8> of encoded
+    /// The `server_params` is provided as a mutable `Vec<u8>` of encoded
```

Comment:
is something new detecting these doc comment formatting issues?

Reply:
I don't think it's anything new. I just think there's nothing currently enforcing them. I ran `cargo doc` to look at the contract docs and noticed all of these warnings so I thought I would fix them while I'm at it.